lib/traject/macros/csv.rb

```diff
@@ -11,5 +11,11 @@ def column(header_or_index, options = {})
         accumulator.concat(result) unless result.first.to_s.empty?
       end
     end
+
+    def numismatic_id
+      lambda do |row, accumulator, _context|
+        accumulator << row['RecordId'].tr('.', '_')
```

Comment:
That sounds good to me @cbeer, and yeah @jcoyne - this was because the dots are throwing routing errors.